### PR TITLE
Migrate text from guideline

### PR DESCRIPTION
**Pull request by ct-clearhaus**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -93,6 +93,15 @@ easy rolling of API keys. A merchant account usually uses one API key.
 API keys can have an expire date. API keys can be enabled or disabled. For
 merchant accounts, these details can be observed in the Clearhaus Dashboard.
 
+An API key points to an account. An account can have multiple API keys. For
```
Comment (mt-clearhaus) [inline source/index.html.md:96]: Very similar to line 89.

Comment (ct-clearhaus) [reply source/index.html.md]: Resolved by 9e24cc12c69ca2c7a5118266cbd5c8512baa1fe4

**Comment by ct-clearhaus** [comment]
Feel free to merge if you are happy 🙂

**Comment by thejspr** [comment]
Is this still relevant? The text seems okay, but not sure if it's still relevant to get into the documentation.

File: source/index.html.md [hunk 2]
```diff
@@ -93,6 +93,13 @@ easy rolling of API keys. A merchant account usually uses one API key.
 API keys can have an expire date. API keys can be enabled or disabled. For
 merchant accounts, these details can be observed in the Clearhaus Dashboard.
 
+For merchant accounts, the use of one active (enabled and unexpired) API key is
+indistinguishable from the use of another active API key. An authorization made
+with one API key can be captured using any active API key for that same merchant
+account.
+This is in contrast to accounts which are independent; an authorization made on
+one merchant account cannot be captured on another account.
```
Comment (mt-clearhaus) [inline source/index.html.md:101]:
```suggestion
For a merchant account, the use of one active (enabled and unexpired) API key
is indistinguishable from the use of another active API key. An authorization
made with one API key can be captured using any active API key for that same
merchant account. In contrast, an authorization made on one merchant account
cannot be captured on another account.
```

Comment (ct-clearhaus) [reply source/index.html.md:101]: Just wrapping, right?

**Review by mt-clearhaus — APPROVED** [review]

**Comment by mt-clearhaus** [comment]
Rendering:

Screenshot from 2024-04-04 13-20-57

**Review by mt-clearhaus — APPROVED** [review]

**Pull request merged**